### PR TITLE
Automate Psychic Amps

### PR DESCRIPTION
**Pull request by TikaelSol**
Need to chase down the the DamageDice REs aren't working to handle the additional heightening effects, but these are otherwise ready. The feat based amps are localized but not handled until we have a way to dynamically build suboptions.

### CODE REVIEW

File: packs/classfeatures/the-distant-grasp.json [hunk 1]
```diff
@@ -24,7 +24,125 @@
...
         },
-        "rules": [],
+        "rules": [
+            {
+                "itemType": "spell",
+                "key": "ItemAlteration",
+                "mode": "add",
+                "predicate": [
+                    {
+                        "or": [
```
Comment (stwlam) [inline packs/classfeatures/the-distant-grasp.json:34]:
```suggestion
                        "spellcasting:category:spontaneous",
                        "spellcasting:tradition:occult",
                        "or": [
```
Best we can do for now to not step on innate/archetype spell collections.

File: packs/classfeatures/the-distant-grasp.json [hunk 2]
```diff
@@ -24,7 +24,127 @@
...
+        "rules": [
+            {
+                "itemType": "spell",
+                "key": "ItemAlteration",
+                "mode": "add",
+                "predicate": [
+                    {
+                        "spellcasting:category:spontaneous",
+                        "spellcasting:tradition:occult",
+                        "or": [
```
Comment (TikaelSol) [inline packs/classfeatures/the-distant-grasp.json:36]:
```suggestion
                        {
                            "or": [
```

File: packs/classfeatures/the-distant-grasp.json [hunk 3]
```diff
@@ -24,7 +24,128 @@
...
             "title": "Pathfinder Dark Archive"
         },
-        "rules": [],
+        "rules": [
+            {
+                "itemType": "spell",
+                "key": "ItemAlteration",
+                "mode": "add",
+                "predicate": [
+                    {
```
Comment (TikaelSol) [inline packs/classfeatures/the-distant-grasp.json:33]:
```suggestion
```

File: packs/classfeatures/conscious-mind.json [hunk 4]
```diff
@@ -39,6 +39,35 @@
...
+                "toggleable": true
+            },
+            {
+                "itemType": "spell",
+                "key": "ItemAlteration",
+                "mode": "add",
+                "predicate": [
+                    "item:trait:psychic"
+                ],
+                "property": "other-tags",
```
Comment (stwlam) [inline packs/classfeatures/conscious-mind.json:58]:
```suggestion
                "priority": 19,
                "property": "other-tags",
```

File: packs/classfeatures/the-distant-grasp.json [hunk 5]
```diff
@@ -24,7 +24,127 @@
...
+                        "spellcasting:category:spontaneous",
+                        "spellcasting:tradition:occult",
+                        {
+                            "or": [
+                            "item:slug:telekinetic-hand",
+                            "item:slug:telekinetic-projectile"
+                        ]
+                    }
+                ],
+                "property": "other-tags",
```
Comment (stwlam) [inline packs/classfeatures/the-distant-grasp.json:42]:
```suggestion
                "priority": 19,
                "property": "other-tags",
```

File: packs/spells/telekinetic-rend.json [hunk 6]
```diff
@@ -19,7 +19,7 @@
...
+            "value": "<p><span style=\"font-family:var(--font-primary);font-size:var(--font-size-14)\">Your mind creates a violent axis of motion in a nearby space. You deal your choice of 1d6 bludgeoning or 1d6 slashing damage to all creatures in the area, with a basic Fortitude save. A creature large enough to be in more than one of the bursts takes damage only once.</span></p>\n<hr />\n<p><strong>Heightened (+2)</strong> The damage increases by 1d6.</p>"
```
Comment (TikaelSol) [inline packs/spells/telekinetic-rend.json:22]:
```suggestion
            "value": "<p>Your mind creates a violent axis of motion in a nearby space. You deal your choice of 1d6 bludgeoning or 1d6 slashing damage to all creatures in the area, with a basic Fortitude save. A creature large enough to be in more than one of the bursts takes damage only once.</p>\n<hr />\n<p><strong>Heightened (+2)</strong> The damage increases by 1d6.</p>"
```

File: static/lang/re-en.json [hunk 7]
```diff
@@ -3341,4 +3379,4 @@
             }
         }
     }
-}
+}
```
Comment (TikaelSol) [inline static/lang/re-en.json:3382]:
```suggestion
}

```

File: packs/classfeatures/the-infinite-eye.json [hunk 8]
```diff
@@ -24,7 +24,136 @@
...
+                "mode": "add",
+                "predicate": [
+                    {
+                        "or": [
+                            "item:slug:detect-magic",
+                            "item:slug:guidance"
+                        ]
+                    }
+                ],
+                "priority": 100,
```
Comment (TikaelSol) [inline packs/classfeatures/the-infinite-eye.json:40]:
```suggestion
                "priority": 19,
```

File: packs/spells/distortion-lens.json [hunk 9]
```diff
@@ -14,7 +14,7 @@
...
+            "value": "<p>You create a magical lens that distorts space as best suits you. You create the lens in a space in range, even suspended in midair. If your or an ally's ranged attack passes through the lens, the attack gains an additional 10 feet of range; if an enemy's ranged attack would pass through the lens, it requires an additional 10 feet of range to move through, though the enemy knows before using its ability whether the interference puts the target out of range. An ally whose space overlaps the lens can increase the range of its ranged attacks, but an enemy whose space overlaps the lens doesn't reduce the range of its ranged attacks.</p>\n<p>The first time each round you Sustain the Spell, you can choose to relocate it to another square within range. The lens disappears if you cast <em>distortion lens</em> again.</p>\n<hr />\n<p><strong>Heightened (+3)</strong> The lens increases or decreases the range of abilities by an additional 5 feet.</p>"
```
[the remark this answers is not in the document]
Comment (TikaelSol) [reply packs/spells/distortion-lens.json:17]: I did it to keep things consistent, and moved the amp description to the conscious mind. That was the extra text and amp heightening can be displayed only when needed.

**Pull request merged**